tests/test_config.py

```diff
@@ -278,3 +278,36 @@ def test_malformatted_toml_raises_error(self, tmp_path):
             formatter.read_black_config(path)
 
         assert error.match("invalid character")
+
+    def test_skip_string_normalisation_handled_with_snakecase(self, tmp_path):
+        line_length = 88
+        formatter = setup_formatter("", line_length=line_length)
+        path = tmp_path / "config.toml"
+        path.write_text("[tool.black]\nskip_string_normalization = false")
+
+        actual = formatter.read_black_config(path)
+        expected = black.FileMode(line_length=line_length, string_normalization=True)
```

Comment:
Again, sure we need to support `skip_string_normalization`. I see it's a black CLI argument but that doesn't mean it should be a valid entry in configfile?

Reply:
It is also a valid entry in the `black` configuration file. Both variations can be used.